### PR TITLE
Add Planar move plugin

### PR DESCRIPTION
**Pull request by SystemDiagnosticss**
Add Planar move plugin for mecanum wheels simulated

### CODE REVIEW

**Comment by SystemDiagnosticss** [comment]
https://youtu.be/Lo1OUz-_Y0g

File: leobot_gazebo/package.xml [hunk 1]
```diff
@@ -13,6 +13,7 @@
   <run_depend>gazebo_ros_control</run_depend>
   <run_depend>leobot_description</run_depend>
 
+
```
Comment (AndriyPt) [inline leobot_gazebo/package.xml:16]: please remove unneeded line

File: leobot_description/robot/leobot.gazebo.xacro [hunk 2]
```diff
@@ -237,4 +237,16 @@
         </sensor>
     </gazebo>
 
+    <gazebo>
+    <plugin name="object_controller" filename="libgazebo_ros_planar_move.so">
+      <commandTopic>/wheel_diff_drive_controller/cmd_vel</commandTopic>
```
Comment (AndriyPt) [inline leobot_description/robot/leobot.gazebo.xacro]: there topic are not needed. The only control topic is specified here https://github.com/ros-ukraine/leobot/blob/kinetic-devel/leobot_control/launch/control.launch#L11
Please read more about http://wiki.ros.org/twist_mux to find out how it works

File: leobot_description/robot/leobot.gazebo.xacro [hunk 3]
```diff
@@ -237,4 +237,16 @@
...
+      <commandTopic>/teleop_twist_keyboard/cmd_vel</commandTopic>
+      <commandTopic>/joy_teleop/cmd_vel</commandTopic>
+      <odometryTopic>/mecanum_velocity_controller/odom</odometryTopic>
+      <odometryFrame>odom</odometryFrame>
+      <odometryRate>10.0</odometryRate>
+      <robotBaseFrame>FL_wheel_link</robotBaseFrame>
```
Comment (AndriyPt) [inline leobot_description/robot/leobot.gazebo.xacro]: Please created dummy link which will slide on the ground and use it as robot base frame. Check this example https://github.com/KristofRobot/frobo/blob/master/frobo_description/urdf/base.urdf.xacro#L200-L213

**Review by AndriyPt — CHANGES_REQUESTED** [review]

**Comment by AndriyPt** [comment]
Closing PR since Planar Plugin was added to kinetic-devel

**Pull request closed without merging**